README.md

```diff
@@ -178,6 +181,77 @@ swap is initiated successfully, `loopd` will see the process through.
+
+Estimated on-chain sweep fee:        149 sat
+Max on-chain sweep fee:              14900 sat
+Max off-chain swap routing fee:      20010 sat
+Max off-chain prepay routing fee:    36 sat
+Max no show penalty (prepay):        1337 sat
+Max swap fee:                        1100 sat
+CONTINUE SWAP? (y/n):
+```
+
+Explanation:
+
+- **Max swap fees for <x> sat Loop Out** (36'046 sat): The absolute maximum in
+  fees that need to be paid. This includes on-chain and off-chain fees. This
+  represents the ceiling or worst-case scenario. The actual fees will likely be
+  lower. This is the sum of `14900 + 20010 + 36 + 1100` (see below).
+- **Estimated on-chain sweep fee** (149 sat): The estimated cost to sweep the
+  HTLC in case of success, calculated based on the _current_ on-chain fees.
+  This value is called `miner_fee` in the gRPC/REST responses.
+- **Max on-chain sweep fee** (14'900 sat): The maximum on-chain fee the daemon
+  is going to allow for sweeping the HTLC in case of success. A fee estimation
+  based on the `--conf_target` flag is always performed before sweeping. The
+  factor of `100` times the estimated fee is applied in case the fees spike
+  between the time the swap is initiated and the time the HTLC can be swept. But
+  that is the absolute worst-case fee that will be paid. If there is no fee
+  spike, a normal, much lower fee will be used.
+- **Max off-chain swap routing fee** (20'010 sat): The maximum off-chain
+  routing fee that the daemon should pay when finding a route to pay the
+  Lightning invoice. This is a hard limit. If no route with a lower or equal fee
+  is found, the payment (and the swap) is aborted. This value is calculated
+  statically based on the swap amount (see `maxRoutingFeeBase` and
+  `maxRoutingFeeRate` in `cmd/loop/main.go`).
+- **Max off-chain prepay routing fee** (36 sat): The maximum off-chain routing
+  fee that the daemon should pay when finding a route to pay the prepay fee.
+  This is a hard limit. If no route with a lower or equal fee is found, the
+  payment is aborted. This value is calculated statically based on the prepay
+  amount (see `maxRoutingFeeBase` and `maxRoutingFeeRate` in `cmd/loop/main.go`).
+- **Max no show penalty (prepay)** (1'337 sat): This is the amount that has to be
+  pre-paid (off-chain) before the server publishes the HTLC on-chain. This is
+  necessary to ensure the server's on-chain fees are paid if the client aborts
+  and never completes the swap _after_ the HTLC has been published on-chain.
+  If the swap completes normally, this amount is counted towards the full swap
+  amount and therefore is actually a pre-payment and not a fee. This value is
+  called `prepay_amt` in the gRPC/REST responses.
+- **Max swap fee** (1'100 sat): The maximum amount of service fees we allow the
+  server to charge for executing the swap. This value must be obtained by asking
+  the server for a quote first. If this doesn't match the quoted fee, the server
```

Comment:
"The value is obtained via a quote" and then "if this (the value obtained via a quote) doesn't match the quoted fee" -> can never happen?

What happens is that the swap is aborted if the fee proposed by the server after initiating the swap exceeds the max